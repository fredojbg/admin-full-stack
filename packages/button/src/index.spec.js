import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { Button, ButtonAzul } from '.';

describe('Button', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Button>{'Test'}</Button>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('ButtonAzul', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<ButtonAzul>{'Test'}</ButtonAzul>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});