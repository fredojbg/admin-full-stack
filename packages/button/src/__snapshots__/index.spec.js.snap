// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
.c0 {
  color: #fff;
  background: red;
  cursor: pointer;
  font-size: 1rem;
  font-weight: 300;
  padding: 9px 36px;
  border-radius: 4px;
}

<button
  className="c0"
>
  Test
</button>
`;

exports[`ButtonAzul renders correctly 1`] = `
.c0 {
  color: #fff;
  background: blue;
  cursor: pointer;
  font-size: 16rem;
  font-weight: 300;
  padding: 19px 16px;
  border-radius: 14px;
}

<button
  className="c0"
>
  Test
</button>
`;
